src/test/java/org/folio/marc4ld/Marc2LdTestBase.java

```diff
@@ -84,6 +98,16 @@ protected ResourceEdge getWorkEdge(Resource resource) {
     return getFirstOutgoingEdge(resource, withPredicateUri("http://bibfra.me/vocab/lite/instantiates"));
   }
 
+  protected List<ResourceEdge> getEdges(Resource resource, ResourceTypeDictionary... resourceTypes) {
```

Comment:
all newly added methods are actually pure Util ones, right? There is no interaction with any field of this class.
Meaning that there is no point to put them into this hierarchical (parent) class, it could be simple Util class, like TestUtil.
So I suggest to extract all such methods out of this class, even previously existed

Reply:
Agree for getEdges() method, but I'm afraid TestUtil will be so big if we start moving all validation related methods there. A lot new of new validations were introduced with Askhat's PR as well , so I'd suggest to keep validation methods as is and move only things like getEdges()

@PBobylev @pkjacob

Reply:
How about this.

1. Create new class `org.folio.marc4ld.authority.AuthorityValidationHelper` and move the following methods to this new class - `validateIdentifierAndLccn` (rename this method to `validateIdentifier`), `validateSubfocusResources`, `validateSubfocus`
2. Create a new class `org.folio.marc4ld.util.ResourceEdgeHelper` and move all the following method to that class - `getOutgoingEdges` (2 overloaded methods), `getFirstOutgoingEdge` (2 overloaded), `getWorkEdge`, `getEdges`, `withPredicateUri`
3. For @askhat-abishev's story, Askhat will create a new class `org.folio.marc4ld.util.LinkingEntriesTestHelper`

Reply:
sounds good for me

Reply:
org.folio.marc4ld.util package is for Util test classes, org.folio.marc4ld.helper should be fine

Reply:
I like PK's idea to separate it and have multiple util (or helper) classes, it's always good to cut big class into few pieces :)
Regarding package name:
I think It's better to put all test-only classes into <base-package.test, in our case it's 'org.folio.marc4ld.test.*'.
We do the same in mod-linked-data

Reply:
agree, fixed.